astropy/utils/compat/optional_deps.py

```diff
@@ -9,10 +9,11 @@
 # TODO: This list is a duplicate of the dependencies in setup.cfg "all", but
 # some of the package names are different from the pip-install name (e.g.,
 # beautifulsoup4 -> bs4).
-_optional_deps = ['asdf', 'asdf_astropy', 'bleach', 'bottleneck', 'bs4', 'bz2', 'h5py',
-                  'html5lib', 'IPython', 'jplephem', 'lxml', 'matplotlib',
-                  'mpmath', 'pandas', 'PIL', 'pytz', 'scipy', 'skyfield',
-                  'sortedcontainers', 'lzma', 'pyarrow']
+_optional_deps = ['asdf', 'asdf_astropy', 'bleach', 'bottleneck', 'bs4', 'bz2',
+                  'fsspec', 'h5py', 'html5lib', 'IPython', 'jplephem',
+                  'lxml', 'matplotlib', 'mpmath', 'pandas', 'PIL', 'pytz',
+                  's3fs', 'scipy', 'skyfield', 'sortedcontainers', 'lzma',
+                  'pyarrow']
```

Comment:
Note: `fsspec` and `s3fs` are added to `_optional_deps`.  This is open for discussion.

Reply:
I'm OK with it.

Reply:
I don't see any problems with it, there are already a ton of other dependencies listed in there.